src/braket/ir/ahs/physical_field.py

```diff
@@ -0,0 +1,36 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from decimal import Decimal
+from typing import List, Union
+
+from pydantic import BaseModel
+
+from braket.ir.ahs.waveform import Waveform
+
+
+class PhysicalField(BaseModel):
```

Comment:
Why PhysicalField vs Field?

Reply:
We tried avoid any possible conflict between AHS schema variable naming and python/pydantic reserved key word.

Reply:
It's not python reserved (as far as I know) : https://docs.python.org/3.8/reference/lexical_analysis.html#keywords

As for pydantic, I'm not sure how much we should avoid conflict with 3rd-party modules, but it seems like it's too cautious.  We are defining a Field in the BDK though - do you think it will be a problem?